websites/A/Aodesu/presence.ts

```diff
@@ -0,0 +1,122 @@
+const presence = new Presence({
+		clientId: "1101840416866844672",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let iFrameVideo: boolean,
+	currentTime: number,
+	duration: number,
+	paused: boolean,
+	playback: boolean,
+	reproductor: string;
+
+presence.on(
+	"iFrameData",
+	(data: {
+		iframeVideo: {
+			duration: number;
+			iFrameVideo: boolean;
+			currTime: number;
+			paused: boolean;
+			reproductor: string;
+		};
+	}) => {
+		playback = data.iframeVideo.duration !== null ? true : false;
```

Comment:
Unnecessary ternary statement.
```suggestion
		playback = data.iframeVideo.duration !== null;
```